STACpopulator/populator_base.py

```diff
@@ -157,3 +159,5 @@ def ingest(self) -> None:
                     update=self.update,
                     session=self._session,
                 )
+                counter += 1
+                LOGGER.info(f"Processed {counter} data items")
```

Comment:
Should there be a warning log if the STAC item was not generated/returned? Possibility an unexpected error that was silently ignored?

Reply:
Yes, good point. I've been wanting to add some more error management code. For the moment, I have added an error log message here.

Reply:
Can you add the item name and loc to the log error message so we can debug the one that failed appropriately?